examples/simple_shield/Prover.toml

```diff
@@ -2,6 +2,6 @@ value = 0
 priv_key = 0
 note_root = 0
 index = 0
-note_hash_path = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+# note_hash_path = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
Will we need this in the future? Otherwise lets remove it

Reply:
Yep, this is there for the actual application use case, so it can be removed